sentences/sv/_common.yaml

```diff
@@ -51,14 +51,19 @@ expansion_rules:
   area: "{area}[e | me][t | ts | n | ns ]"
   vad: "(vad är | vad är det | vad är det för | vilken)"
   är: "(är det | är)"
-  brightness: "{brightness} [percent]"
+  brightness: "{brightness} [procent]"
   slå_på: "(tänd | sätt (på | igång | fart på) | slå (på | till) | starta)"
   slå_av: "(slå (av | från | ifrån) | släck | stäng av | stoppa)"
   i_på: "(i | på | vid | bredvid)"
   temp: "(temperatur)"
   alla: "(all | alla | allt)"
-  ljusintensitet: "(ljusstyrka | intensitet | styrka | nivå | effekt)"
-  ljuskällor: "(ljuskällor[na] | lampor[na] | lampa[n]| belysning[en] | ljus[en|et] | glödlampor[na] | lysrör[en|et])"
+  möjliga: "(möjlig[a] | tillgänglig[a])"
+  ljusintensitet: "(ljus[et] | belysning[en] | [ljus](styrka | intensitet | nivå | effekt))"
+  lampa: "(ljuset | belysning[en] | lampan | lampor[na])"
+  ljuskällor: "(ljuskällor[na] | belysning[en] | ljus[en | et] | [glöd|led|diod]lamp[ a | an | or | orna] | lysrör[en| et] | belysning[en])"
+  dimra: "(sätt | vrid | dra [på] | skruva [på] | dimra | dimma | ställ [in] | höj | sänk)"
+  maximal: "(maximal | max[imum] | högsta | starkaste | intensivaste | full)"
```

Comment:
Not sure but maybe "`full"` also should cover `"full[t]"`

Reply:
Added.